sway-lib-std/src/call_frames.sw

```diff
@@ -20,13 +20,47 @@ const SECOND_PARAMETER_OFFSET: u64 = 74;
 //  Accessing the current call frame
 //
 /// Get the current contract's id when called in an internal context.
+///
+/// # Additional Information
+///
 /// **_Note:_** If called in an external context, this will **not** return a contract ID.
-// @dev If called externally, will actually return a pointer to the transaction ID.
+/// If called externally, will actually return a pointer to the transaction ID.
+/// 
+/// # Returns
+///
+/// * [ContractId] - The contract id of this contract.
+///
+/// # Examples
+///
+/// ```sway
+/// use std::call_frames::contract_id;
+///
+/// fn foo() {
+///     let this_contract = contract_id();
```

Comment:
example seems too barebones, maybe mint and then transfer some tokens by using the this_contract as the assetid

Reply:
I wasn't sure what else I can add here that is actually relevant to the function.

I also don't think an example with minting tokens and using the contract Id as the asset id is a good idea since we are changing this with the multi token implementation coming soon™

Reply:
When multi token comes out it would only change a line wouldnt it? I dont see why that would be a reason to not have that example right now, we can change it later